### DISCUSSION TITLE
Stunning blow

### DISCUSSION THREAD

**Discussion by Lambosaurus** [Lambosaurus/hives-and-torches#36] [Feats]
When wielding hammers, you gain a new critical effect stunning blow.
The foe is stunned until the end of their next turn.

Stunned:
When a character is stunned they:
 - May not take reactions
 - May not take their primary actions. (Or should it be only secondary action? This pretty much controls whether they can attack.)
 

We can reuse stunned as a status.

**Comment by LouderthanThunder** [comment]
Stunning should be hard. Two crits stuns one removes an action 

Seperate it into daze and stun 

In my opinion

**Reply by Lambosaurus** [reply]
So dazed would just remove reactions and your secondary action?
Stun would just skip your turn?
